### ISSUE TITLE
Epic: Combine demo and pitch videos into a single pitch video [20,20,1]

### ISSUE THREAD

**Issue by stenington** [Iridescent-CM/technovation-app#2191]
This ticket comes from #2177 

In scope for this ticket:
* Everywhere the demo and pitch video appear (screenshots below), we would now only show "Pitch video" which will actually be a combo of the two in terms of content.
* Remove demo video as a required field for app completion
* Rename the "demo video" element in the scoring rubric to "usability" or something similar (this change would NOT be visible to any user)
* Deal with any other appearance of "demo video" in the code as appropriate

This ticket relies on #2187 to clear out past season submissions so we don't have to worry about the schema change messing up historical data.

## Screenshots
**Admin/RA**
![Screen Shot 2019-09-05 at 10 26 03 AM](https://user-images.githubusercontent.com/55319/64355252-e133ed00-cfce-11e9-81e6-8350056aef11.png)
![Screen Shot 2019-09-05 at 10 44 49 AM](https://user-images.githubusercontent.com/55319/64355264-e2651a00-cfce-11e9-9b5a-00642b81dd09.png)

**Student and Mentor**
![Screen Shot 2019-09-05 at 10 30 05 AM](https://user-images.githubusercontent.com/55319/64355253-e1cc8380-cfce-11e9-88e4-f6e980c41bc2.png)
![Screen Shot 2019-09-05 at 10 30 13 AM](https://user-images.githubusercontent.com/55319/64355255-e1cc8380-cfce-11e9-9f1d-90bbe4ff18b1.png)
![Screen Shot 2019-09-05 at 10 30 17 AM](https://user-images.githubusercontent.com/55319/64355257-e1cc8380-cfce-11e9-8efe-afd9d697e0dd.png)
![Screen Shot 2019-09-05 at 10 30 46 AM](https://user-images.githubusercontent.com/55319/64355258-e1cc8380-cfce-11e9-83ae-77b780c41406.png)
![Screen Shot 2019-09-05 at 10 41 30 AM](https://user-images.githubusercontent.com/55319/64355259-e1cc8380-cfce-11e9-8fc0-d8931587ec3f.png)

**Judge**

![Screen Shot 2019-09-05 at 10 42 18 AM](https://user-images.githubusercontent.com/55319/64355261-e1cc8380-cfce-11e9-8916-fd3b1d8331b5.png)
![Screen Shot 2019-09-05 at 10 42 31 AM](https://user-images.githubusercontent.com/55319/64355262-e2651a00-cfce-11e9-9c02-9d0da222f901.png)

**Comment by hellafitz** [comment]
QA may be time-intensive due to the number of user types whose experience needs to be reviewed. Potential tactics: prioritize w/ Devin and Amy, set calendar events, invite more QA people to review in parallel as different user types.

**Comment by stenington** [comment]
In order to break this down into smaller steps, I suppose we could try something like:
* Remove demo video as a required part of submission (even if it's still in UI)
* Remove demo video from admin
  * Team submission details/edit page
  * Submissions page, more columns
* Remove demo video from Student/Mentor UI
  * Submission checklist
  * Submission section
  * Published submission
* Remove from judge UI
  * Review submission section
  * Ideation section
* Optional final clear-out
  * migration to remove it from DB
  * if any libraries or config no longer needed, remove
  * rake tasks or other stray mentions

**Comment by stenington** [comment]
Broken into #2252, #2253, #2254, #2255, #2256
